tests/regression/test_slepc.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import numpy as np
+from firedrake import *
+from firedrake.petsc import PETSc
+
+
+def topetsc(A):
+    return A.M.handle
+
+
+def test_import():
+    from slepc4py import SLEPc
+
+
+def test_laplace_physical_ev(parallel=False):
+    from slepc4py import SLEPc
```

Comment:
Since slepc is an optional dep, can you guard this test like so:

```python
try:
    from slepc4py import SLEPc
except ImportError:
    pytest.skip(msg="SLEPc unavailable, skipping eigenvalue test")
```